lib/jazzy/config.rb

```diff
@@ -400,6 +401,12 @@ def print_attr_description(attr)
       end
     end
 
+    class PathnameArray < Array
```

Comment:
ah. I think maybe just adding a `path` attr to the config options might make sense there. That subclass is a code smell to me, it's saying we're trying to do too much magic instead of explicitly stating what should be done